client/lib/analytics/track-resurrections/test/index.jsx

```diff
@@ -0,0 +1,78 @@
+/**
+ * @jest-environment jsdom
+ */
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { render } from '@testing-library/react';
+import * as reactRedux from 'react-redux';
+import TrackResurrections from '../';
+
+jest.mock( 'react-redux', () => ( {
+	useSelector: jest.fn(),
+} ) );
```

Comment:
Using `renderWithProvider` might help to avoid mocking `react-redux` and some of the extra testing boilerplate we have in test setup and teardown:

https://github.com/Automattic/wp-calypso/blob/4910db94dde21aced9111956d87aae3389df408d/client/test-helpers/testing-library/index.js#L9

Example usage:

https://github.com/Automattic/wp-calypso/blob/5af74e2eae7ac275ba03221cb069f461ac9e122b/client/blocks/daily-post-button/test/index.jsx#L24

Reply:
Thanks, let me explore that.

Reply:
Updated code to use renderWithProvider.